_posts/2020-02-11-agrc-purchase-of-hexagon-imagery.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'AGRC Purchase of Hexagon Imagery'
+author:
+  display_name: Matt Peters
+  email: mpeters@utah.gov
+date: 2020-02-11 16:15:02
+categories:
+  - Featured
+tags:
+  - imagery
+  - datums
+  - aerial photography
+---
+AGRC recently purchased a license to Hexagon’s statewide high resolution aerial photography (6” and 12” pixels) on behalf of Utah state, regional, and local government organizations. Since the purchase, AGRC staff have been working to provide a seamless delivery of the product through the Discover system, like we did with the Google Imagery. The licensing terms of the Hexagon product will remain the same as the terms for the Google Imagery. You can find information on accessing both the Google Imagery and the new Hexagon product on the [AGRC website]({% link discover/index.html %}).
```

Comment:
```suggestion
AGRC recently purchased a license to Hexagon’s statewide high resolution aerial photography (6” and 12” pixels) on behalf of Utah state, regional, and local government organizations. Since the purchase, AGRC staff have been working to provide a seamless delivery of the product through the Discover system, like we did with the Google Imagery. The licensing terms of the Hexagon product will remain the same as the terms for the Google Imagery. You can find information on accessing both the Google Imagery and the new Hexagon product on the [Discover page]({% link discover/index.html %}).
```